Makefile

```diff
@@ -422,6 +422,7 @@ subsystem-test: test
 subsystem-test-kube-api: test-kube-api
 
 _run_subsystem_test:
+	ENABLE_SUBSYSTEM_TESTS=true \
```

Comment:
ENABLE_SUBSYSTEM_TESTS=true makes the subsystem tests not skip themselves, that's all there is to it.

It allows you to simply run `go test ./...` or `gotestsum` in the main directory to run unit-tests, without the subsystem tests trying to run themselves.

The reason they're excluded is because subsystem tests require you to set up dedicated infrastructure outside, while unit tests set up the database on their own

Reply:
Ah, as long as ``ENABLE_SUBSYSTEM_TESTS=true`` doesn't (by itself) skip unit-tests I'm okay with this
Just needed clarification because I couldn't spot in the code the actual behavior